File: 242-valid-anagram/242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        vector<int> freq(26, 0);
        for(auto c: s) freq[c - 'a']++;
        for(auto c: t) freq[c - 'a']--;
        for(auto f: freq) {
            if(f != 0) return false;
        }
        return true;
    }
};

File: 201-300/242-valid-anagram/242-valid-anagram.java
public class Solution {
    public boolean isAnagram(String s, String t) {
    	char[] sa = s.toCharArray();
    	char[] ta = t.toCharArray();
    	Arrays.sort(sa);
    	Arrays.sort(ta);
        return new String(sa).equals(new String(ta));
    }
}